src/main/java/study/step4/domain/LottoQuantity.java

```diff
@@ -0,0 +1,31 @@
+package study.step4.domain;
+
+public class LottoQuantity {
+    private static final int LOTTO_AMOUNT = 1000;
+    private static final String BUY_AMOUNT_ERROR = "천원 이상 구매해주시길 바랍니다.";
+
+    private final int quantity;
+
+    public LottoQuantity(int buyAmount, int manualLottoQuantity) {
+        this.quantity = amountToQuantity(buyAmount) - manualLottoQuantity;
+    }
+
+    public LottoQuantity(int buyAmount) {
+        this.quantity = amountToQuantity(buyAmount);
+    }
```

Comment:
매개변수가 적은 생성자를 더 위로 올리는게 가독성이 높습니다. 
또한 이런 경우 점진적 생성자 패턴을 사용할 수 있죠. 
```suggestion
    public LottoQuantity(int buyAmount) {
        this(buyAmount, 0);
    }
```